js/iframe-coordinator/libs/BackgroundClient.ts

```diff
@@ -0,0 +1,35 @@
+import { ToastingClient, ToastOptions } from './types';
+import {
+  WORKER_MESSAGING_PROTOCOL_NAME,
+  WorkerToHostMessageTypes
+} from './workers/constants';
+
+const ctx: Worker = self as any;
+
+export default class BackgroundClient implements ToastingClient {
+  public requestToast(
+    message: string,
+    { title = null, custom = null }: ToastOptions = {}
+  ): void {
+    this._publishMessageToHost(WorkerToHostMessageTypes.ToastRequest, {
+      title,
+      message,
+      custom
+    });
+  }
+
+  // TODO Change this to URL based?
```

Comment:
@MattCheely - we talked about converting this to a straight URL.  Should I keep it as a route for now?